### ISSUE TITLE
Input ability Denied

### ISSUE THREAD

**Issue by SnowJob** [bk138/droidVNC-NG#18]
If you'd like to put out an incentive for fixing this bug, you can do so at https://issuehunt.io/r/bk138/droidVNC-NG

**Describe the bug**
 I am able to launch server on my Samsung A7 tablet.  However I cannot get permission to input data remotely.  I allowed every accessibility except for voice but I seem to always be denied input.  I have been able to enable Scren capturiing and file access

**To Reproduce**

**Expected Behavior**
 I had expected to be able to click on buttons.  However, the remoted mouse icon moves as expected but is unable to activate functions be cause input has been denied by the server.

**Logs/Backtraces**

**Your environment (please complete the following information):**
 - OS and version: Android 9
 - VNC viewer and version: RealVNC 3.7.1.44443

**Additional context**

**Comment by bk138** [comment]
There seems to be a problem with your particular device and/or Android version. Does the app alphavnc work for you?

**Comment by SnowJob** [comment]
My apologies!  Now the installed services showa both you and alpha VNC.  It now works!! Thank you.

Larry

Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by bk138** [comment]
Nothing we can do about Samsung's wording :shrug:

**Comment by polluks2** [comment]
I have the same issue with a Samsung A6 tablet.
Please enable the input! I don't want to change to alpha vnc.
